packages/client/src/runtime/core/engines/data-proxy/DataProxyEngine.ts

```diff
@@ -474,18 +475,18 @@ export class DataProxyEngine extends Engine<DataProxyTxInfoPayload> {
     try {
       url = new URL(serviceURL)
     } catch {
-      throw new InvalidDatasourceError('Could not parse URL of the datasource', {
-        clientVersion: this.clientVersion,
-      })
+      throw new InvalidDatasourceError('Could not parse URL of the datasource', errorInfo)
```

Comment:
Maybe a good opportunity to improve this a bit? This would give a hint that the expected url should start with prisma protocol
```suggestion
      throw new InvalidDatasourceError('Could not parse URL of the datasource. It must be a valid URL that starts with the "prisma://" protocol', errorInfo)
```

Reply:
Made these on par with the regular engine errors.